timemachine/potentials/nonbonded.py

```diff
@@ -155,6 +159,56 @@ def nonbonded_block(xi, xj, box, params_i, params_j, beta, cutoff):
     return jnp.sum(nonbonded_block_unsummed(xi, xj, box, params_i, params_j, beta, cutoff))
 
 
+import itertools
+
+
+def nonbonded_block_rescaled(xi, xj, box, params_i, params_j, beta, cutoff, anchors_i, groups_i):
```

Comment:
nit: is this currently a different interface than is exposed by the GPU potential? (don't immediately see the concept of "groups" there)

Reply:
stale - added a standard API and wrapped inside `potentials.py`